tests/test_url_conversions.py

```diff
@@ -51,6 +52,17 @@ def test_roundtrip_rel(self):
         # should have reconstructed it by this point
         self.assertEqual(os.path.normpath(result), MEDIA_EXAMPLE_PATH_REL)
 
+    def test_premiere_xml_urls(self):
```

Comment:
I was just wondering if it would be possible to test this without introducing the xml file?
Could we pass strings instead?
The reason I'm asking is that we'll be removing all those sample files as a part of the adapter extraction.

Reply:
Sure, could totally switch to using strings for unit tests! That's how I tested this code anyway, by copying examples from `.xml` and `.aaf` files to figure out how the differences in approach.